platform/security/security-oidc-realm/src/main/java/org/codice/ddf/security/oidc/realm/CustomOidcProfileCreator.java

```diff
@@ -47,85 +35,44 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(CustomOidcProfileCreator.class);
 
-  private OIDCProviderMetadata metadata;
-
-  public CustomOidcProfileCreator(
-      OidcConfiguration configuration, OIDCProviderMetadata oidcProviderMetadata) {
+  public CustomOidcProfileCreator(OidcConfiguration configuration) {
     super(configuration);
-    metadata = oidcProviderMetadata;
   }
 
   @Override
   public U create(OidcCredentials credentials, WebContext context) {
     init();
 
-    final AccessToken accessToken = credentials.getAccessToken();
-    final JWT idToken = credentials.getIdToken();
-
     final U profile = getProfileDefinition().newProfile();
-    profile.setAccessToken(accessToken);
-    profile.setIdTokenString(idToken.getParsedString());
 
-    try {
-      IDTokenClaimsSet claimsSet = new IDTokenClaimsSet(idToken.getJWTClaimsSet());
-      assertNotNull("claimsSet", claimsSet);
-      profile.setId(ProfileHelper.sanitizeIdentifier(profile, claimsSet.getSubject()));
-    } catch (java.text.ParseException | ParseException e) {
-      LOGGER.warn("Unable to extract ID token claim set.", e);
+    final AccessToken accessToken = credentials.getAccessToken();
+    if (accessToken != null && !accessToken.getValue().isEmpty()) {
+      profile.setAccessToken(accessToken);
     }
 
     final RefreshToken refreshToken = credentials.getRefreshToken();
     if (refreshToken != null && !refreshToken.getValue().isEmpty()) {
       profile.setRefreshToken(refreshToken);
-      LOGGER.debug("Refresh Token successful retrieved");
     }
 
-    try {
```

Comment:
📘 This was only used to fetch the user info, which we will already have by this point.